### PR TITLE
website improvements

### LINKED ISSUE #281: v3 missing Brave in Browsers section
"Browsers" section at the bottom of https://ipfs.tech/developers/ suggestions:

**(1)** Opera and Brave have native support and not extension so maybe replace "extensions" with "web browser solutions" ?

**(2)**  is missing Brave:

>  

I think an easy fix is to do what we already do for Opera, and link to relevant blogpost (https://brave.com/ipfs-support/).
Brave logo asset is  at https://brave.com/brave-branding-assets/

 **(3)** Browsers  and extensions are more user-facing, and not Developer tools. Could we move "Browsers" section from https://ipfs.tech/developers/ to https://ipfs.tech/ right under "Get Started"  section? It is probably the most useful info for regular users who want to browse websites hosted on IPFS, and we bury it under "Developers" for some reason. 

cc @cwaring  thoughts?

### LINKED ISSUE #283: Estuary is discontinued
## Problem

https://estuary.tech/ is mentioned on /developers page:
https://github.com/ipfs/ipfs-website/blob/aef2321ac1d3c22559d378d5eee54f2c937fb193/pages/developers.vue#L137-L138
but it got discontinued 8/7/23. 

>  

## Solution

@cwaring  perhaps we should avoid endorsing specific service here, and instead link to relevant docs?
(replace Estuary with _"[..] projects use [pinning services](https://docs.ipfs.tech/concepts/persistence/#pinning-services) to [..]"_)

### PR DESCRIPTION
**Pull request by 2color**
- update newsletter tag in blog
- update page title and description to be consistent
- replace estuary with more general pinning services
- fix link to implementations list from the docs
- update CTAs to the new ipfs docs quickstarts

resolves #283, resolves #281

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -66,8 +66,8 @@ export default defineNuxtConfig({
     public: {
       // seoKit
       siteUrl: PUBLIC_SITE_URL,
-      siteName: 'IPFS Powers the Distributed Web',
-      siteDescription: 'The InterPlanetary File System is a peer-to-peer hypermedia protocol designed to preserve and grow humanity\'s knowledge by making the web upgradeable, resilient, and more open.',
+      siteName: 'IPFS is an open system to manage data without a central server',
```
Comment (cwaring) [inline nuxt.config.ts:69]: This key drives for the primary <title> tag for the site used on all pages so I would suggest something shorter, perhaps even simply `IPFS` (however I do prefer the current descriptive title):

2023-09-22 @ 22 43 15@2x

**Review by cwaring — CHANGES_REQUESTED** [review]

File: nuxt.config.ts (hunk 1 above)
Comment (2color) [reply nuxt.config.ts:69]: What do you think about:

"IPFS – open system to manage data without a central server"

That's 58 characters which won't get truncated and still follows the under 60 characters SEO rule of thumb.

[source](https://moz.com/learn/seo/title-tag#:~:text=Keep%20title%20tags%20between%2051,or%20rewrite%20it%20for%20display.)

Comment (2color) [reply nuxt.config.ts:69]: What about taking this change only for the homepage and on the other pages just do: `Developers | IPFS"?

Comment (2color) [reply nuxt.config.ts:69]: Any thoughts? @cwaring @lidel

Comment (cwaring) [reply nuxt.config.ts:69]: If we go for a different title on the index page and sub pages then the index title would end up looking something like this: `An open system to manage data without a central server | IPFS`

Comment (cwaring) [reply nuxt.config.ts:69]: If that sounds ideal happy to make the change now

Comment (2color) [reply nuxt.config.ts:69]: Sounds good -- let's go with that

**Review by cwaring — APPROVED** [review]

File: nuxt.config.ts (hunk 1 above)
Comment (cwaring) [reply nuxt.config.ts]: updated in 724256b

**Pull request merged**